resources/views/hardware/view.blade.php

```diff
@@ -595,10 +595,12 @@
                                                 {{ $asset->warranty_months }}
                                                 {{ trans('admin/hardware/form.months') }}
 
-                                                @if (($asset->serial && $asset->model->manufacturer) && $asset->model->manufacturer->name == 'Apple')
+                                                @if ($asset->serial)
```

Comment:
It's probably not ideal but it's possible that an asset model doesn't have a manufacturer set in which case this page would cause an error ("trying to read property name on null" on line 599 because `$asset->model->manufacturer`  would be null in that case).

Keeping the conditional should handle that (but please test it to make sure):
```suggestion
                                                @if ($asset->serial && $asset->model->manufacturer)
```

Reply:
hmm, weird, that shouldn't have bounced liek that. maybe i hit one more extra ctrl+z without realizing it while cleaning it up. (after I had tested it)

Reply:
Also could have been, since Manufacturer is a required field, I removed it as it was unneeded? but that doesn't feel like something i would have done off hand. Though it could have been.